src/components/Auth/AuthButton.js

```diff
@@ -22,15 +22,19 @@ import React from 'react';
 import { Link } from 'react-router-dom';
 import './AuthButton.css';
 
-export const AuthButton = () => {
+const titleForAuthenticationState = isAuthenticated => (isAuthenticated ? 'Logout' : 'Login');
+
+const linkForAuthenticationState = isAuthenticated => (isAuthenticated ? '/logout' : '/login');
```

Comment:
its always better to move pure helper functions like that outside of a component. good job